scanpipe/pipes/d2d.py

```diff
@@ -500,3 +551,198 @@ def match_purldb(project, extensions, logger=None):
             start_time=start_time,
         )
         _match_purldb_resource(project, to_resource)
+
+
+def map_javascript(project, logger=None):
```

Comment:
> IMO that won’t change the outcome and will needlessly complicate the logic. Since in almost all case we have one-to-many relation ( single source file is generating upto 7-8 different file), in such case if we start by `to` side we will need to supply another layer of logic to selectively get those specific 7-8 file that might relate to a particular file from `from` side.

Reply:
Fair enough, maybe add this explanation in the docstring of the map_javascript ;)